.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   parserOptions: {
     ecmaVersion: "latest",
     sourceType: "module",
-    project: "./tsconfig.json",
+    project: "./tsconfig.lint.json",
```

Comment:
explain this?

Reply:
Specific linting tests require running the TSC compiler. We must select a TSC config that builds both the `test` and `src` directories to get linting.

The `tsconfig.lint.json` is a specific linting configuration in TSC that compiles both `src` and `test`. It's a new file because we don't want this configuration during buildtime.